<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>projectClass</class>
 <widget class="QMainWindow" name="projectClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>768</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>project</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="6,4">
    <item row="0" column="1">
     <widget class="QGroupBox" name="groupBoxData">
      <property name="title">
       <string>信息数据</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="tabPosition">
          <enum>QTabWidget::South</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Triangular</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tabRealTimeStatistics">
          <attribute name="title">
           <string>实时统计</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tabDataSetting">
          <attribute name="title">
           <string>数据设置</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="tabHistoricalStatistics">
          <attribute name="title">
           <string>历史统计</string>
          </attribute>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QGroupBox" name="groupBoxError">
      <property name="title">
       <string>异常</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <widget class="QListWidget" name="listWidgetError">
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="flow">
          <enum>QListView::TopToBottom</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBoxCamera">
      <property name="title">
       <string>相机</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="8,2">
       <item row="0" column="0">
        <widget class="QLabel" name="labelCameraDisplay">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">background:gray</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QListWidget" name="listWidgetCameras">
         <property name="frameShape">
          <enum>QFrame::WinPanel</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="flow">
          <enum>QListView::LeftToRight</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuImageSetting">
    <property name="title">
     <string>图像设置</string>
    </property>
    <addaction name="actionImageSetting"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>历史记录</string>
    </property>
    <addaction name="actionLog"/>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuRun">
    <property name="title">
     <string>运行</string>
    </property>
   </widget>
   <addaction name="menu"/>
   <addaction name="menuImageSetting"/>
   <addaction name="menuRun"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSignal"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionLog">
   <property name="text">
    <string>Log</string>
   </property>
  </action>
  <action name="actionImageSetting">
   <property name="text">
    <string>ImageSetting</string>
   </property>
  </action>
  <action name="actionSignal">
   <property name="icon">
    <iconset>
     <normaloff>Icon/Signal.png</normaloff>Icon/Signal.png</iconset>
   </property>
   <property name="text">
    <string>单次触发</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MyView.qrc"/>
 </resources>
 <connections/>
</ui>
